### ISSUE TITLE
Images render as "blurry" and automatically resized. 

### ISSUE THREAD

**Issue by 223SK** [jmoenig/Snap--Build-Your-Own-Blocks#1037]
When my students import a background that was larger than the stage, the background renders as "blurry." I think it's automatically resizing to the width? She imported an image that 960px wide. Here's [the project](http://snap.berkeley.edu/snapsource/snap.html#present:Username=mommina%20khalid&ProjectName=Piskel).

Thanks for your help.

**Comment by jmoenig** [comment]
Yes, costumes are automatically resized to fit whichever are the stage's bounds at 100% size.

(you can cheat by manually setting the stage to a bigger size in the settings menu before importing a costume, and then setting it back to the desired dimensions after you've imported it ^^)

**Comment by 223SK** [comment]
Awesome, thanks. I'll let them know tomorrow. Thanks.

**Comment by nataliefreed** [comment]
Sorry for bump but I can't find another issue relating to this. For me this is actually cropping stage background images when their dimensions don't match the stage. Specifically when the image is imported it comes in at the full width of the stage, leaving white space at the top and bottom (which is what I wanted). After I save and reopen the project, the image has expanded to fill the stage, cropping off part of the left and right edges of the image. Is this expected behavior? A separate issue?
